S2VX.Game/Editor/S2VXEditor.cs

```diff
@@ -52,6 +52,9 @@ public class S2VXEditor : CompositeDrawable {
         private AudioManager Audio { get; set; }
         public DrawableTrack Track { get; private set; }
 
+        public int SnapDivisor { get; private set; } = 1;
+        private const int MaxSnapDivisor = 4;
```

Comment:
I think having a few more levels would be good, i.e. up to MaxSnapDivisor 16. May be a bit too excessive, but I think it's good to give finer control.